### DISCUSSION TITLE
DietPi-Software | Brave Web Browser

### DISCUSSION THREAD

**Discussion by flnEGknEF** [MichaIng/DietPi#6840] [Software Request]
Both have the --no-ssndbox error and won't start by manual install. I'd really like to see either (preferably both for choice) added to the official dietpi software selection. Fantastic for privacy minded people and there's a "sync chain" setting in I believe both to sync information across desktop and mobile without using those pesky credentials to link devices together (like chrome or Firefox).

**Comment by MichaIng** [comment]
Vivaldi request is here: #6841
Let's make this one for Brave explicitly.

What do you mean with `--no-sandbox` error? This flag is required for Chromium-based browsers when you want to run them as root user. Otherwise it should work without it.

**Reply by flnEGknEF** [reply]
Don't close this one, I'll get the output replied here tomorrow.

**Reply by flnEGknEF** [reply]
After running:
```sh
sudo apt install curl

sudo curl -fsSLo /usr/share/keyrings/brave-browser-beta-archive-keyring.gpg https://brave-browser-apt-beta.s3.brave.com/brave-browser-beta-archive-keyring.gpg

echo "deb [signed-by=/usr/share/keyrings/brave-browser-beta-archive-keyring.gpg] https://brave-browser-apt-beta.s3.brave.com/ stable main"|sudo tee /etc/apt/sources.list.d/brave-browser-beta.list

sudo apt update

sudo apt install brave-browser-beta
```

I receive:
```
[73287:73287:0113/233750.001955:ERROR:zygote_host_impl_linux.cc(100)] Running as root without --no-sandbox is not supported. See https://crbug.com/638180.
```

**Reply by MichaIng** [reply]
As I said, you need to run it with `--no-sandbox`, or better, run the desktop/brave not as root user but as regular non-root login user, like the `dietpi` user or an own one.

**Reply by flnEGknEF** [reply]
I'm not as experienced with Linux here, so what do you mean "run it with --no-ssndbox"?

**Reply by MichaIng** [reply]
Try to run `brave --sandbox` from console. I am not sure where its config files are located, probably in `/etc/brave` or `/etc/brave.d` or similar. There you can put or edit a config file which contains the flags. In Chromium, it needs to contain a line like
```sh
export CHROMIUM_FLAGS="$CHROMIUM_FLAGS --sandbox"
```
Since Brave is Chromium-based, it should be similar.

But really, there is a reason why this does not work OOTB: Use a non-root user for desktop sessions.
